assets/migrations/scripts/20200531094935_add_version_to_location_metadata.sql

```diff
@@ -0,0 +1,32 @@
+--
+--    Copyright 2010-2016 the original author or authors.
+--
+--    Licensed under the Apache License, Version 2.0 (the "License");
+--    you may not use this file except in compliance with the License.
+--    You may obtain a copy of the License at
+--
+--       http://www.apache.org/licenses/LICENSE-2.0
+--
+--    Unless required by applicable law or agreed to in writing, software
+--    distributed under the License is distributed on an "AS IS" BASIS,
+--    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--    See the License for the specific language governing permissions and
+--    limitations under the License.
+--
+
+-- // add version to location metadata
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE core.location_metadata Add column version integer NOT NULL DEFAULT 0;
+
+ALTER TABLE core.location_metadata DROP CONSTRAINT IF EXISTS location_metadata_geojson_id_key;
+ALTER TABLE core.location_metadata ADD CONSTRAINT location_metadata_geojson_id_version_indx UNIQUE (geojson_id, version);
```

Comment:
Add a query for populating version for existing locations to `location_metadata.version` column from the location table

Reply:
Resolved by https://github.com/OpenSRP/opensrp-server-configs/pull/75